pkg/scheduler/apis/config/validation/validation.go

```diff
@@ -72,6 +77,45 @@ func ValidateKubeSchedulerLeaderElectionConfiguration(cc *config.KubeSchedulerLe
 	return allErrs
 }
 
+func validateTopologySpreadConstraints(constraints []v1.TopologySpreadConstraint, fldPath *field.Path) field.ErrorList {
+	var allErrs field.ErrorList
+	for i, c := range constraints {
+		p := fldPath.Index(i)
+		if c.MaxSkew <= 0 {
+			f := p.Child("maxSkew")
+			allErrs = append(allErrs, field.Invalid(f, c.MaxSkew, "must be greater than zero"))
+		}
+		if len(c.TopologyKey) == 0 {
```

Comment:
do we need to validate TopologyKey is a valid label key?

Reply:
Done